static/_redirects

```diff
@@ -275,3 +275,7 @@ https://0-12-0.docs.pomerium.io/ https://0-12-0.docs.pomerium.com/:splat 301!
 https://0-11-0.docs.pomerium.io/ https://0-11-0.docs.pomerium.com/:splat 301!
 https://0-10-0.docs.pomerium.io/ https://0-10-0.docs.pomerium.com/:splat 301!
 
+# 301 deprecated guides in v21 to v20 guides
+https://0-21-0.docs.pomerium.io/docs/guides/nginx https://0-20-0.docs.pomerium.com/docs/guides/nginx 301!
+https://0-21-0.docs.pomerium.io/docs/guides/traefik-ingress https://0-20-0.docs.pomerium.com/docs/guides/traefik-ingress 301!
```

Comment:
maybe it should be 

```suggestion
/docs/guides/nginx https://0-20-0.docs.pomerium.com/docs/guides/nginx 301!
/docs/guides/traefik-ingress https://0-20-0.docs.pomerium.com/docs/guides/traefik-ingress 301!
```

Reply:
Agree; we should redirect for the latest deployed.